specs/commandline/key.md

```diff
@@ -45,10 +45,10 @@ Flags:
 ### notation key delete
 
 ```text
-Delete key from signing key list
+Remove key from signing key list
 
 Usage:
-  notation key delete [flags] <key_name>...
+  notation key remove [flags] <key_name>...
```

Comment:
Do we want to have an alias such as
```
Usage:
  notation key remove [flags] <key_name>...

Aliases:
  remove, delete
```

Reply:
I would not suggest adding an alias here since `Delete` has a different meaning from `Remove`. `Remove` is more accurate in the context of `notation key`. 

I am open to this question. @toddysm @iamsamirzon @yizha1 @vaninrao10

Reply:
My opinion is No. So there are no aliases for `notation key remove` command

Reply:
I believe, we had this discussion with either @iamsamirzon or @priteshbandi in the past. We need to be clear what is the meaning of `delete` and `remove`. It seems different people have different expectation. As a non-English speaker my expectation is similar to @shizhMSFT, i.e. I expect the key (and associated assets) to be gone completely. Though, the implementation right now is that the key is only "deleted" from the list of keys but not removed from the file system and associated certs are also not removed. As an experience, I would expect the command to be intelligent enough to give choices to do all of the above and not have separate command to "remove".

*Note:* We are talking about `key delete` here as far as I can see from @shizhMSFT comment.

Reply:
`remove` v.s.` delete`:
- `remove` is about removing something from a place and **can** be easily recovered. For instance, remove a song from a playlist.
- `delete` is about removing something from a place and **cannot** be easily recovered. For instance, delete a song from a music library.

In `notation key`, all keys are referenced. Therefore, `key remove` is more suitable. Removing the key from list will not delete the key from the storage.
In `notation cert`, all certs are stored as files. Therefore, `cert delete` is more suitable. Deleting the cert will delete the cert file from the disk and cannot be easily recovered.

However, as @toddysm mentioned, non-English speakers mix the term `remove` and `delete`. That's why I initially proposed to add `delete` as an alias to `remove`. Of course, we need to clearly state the key is **removed** in the response.

On the other hand, completely removing the key using `notation` is technically not possible unless a `delete key` protocol is introduced for plugins.

Reply:
My two cents:

It seems a practice that `add` and `remove` are a pair; `create` and `delete` are a pair. This is also another reason `notation key` and `notation cert` commands cause confusion.

Since we don't create any resources (key and certificate) except the `notation cert generate-test`. Using `add` and `remove` are more appropriate in most scenarios for `notation key` and `notation cert` commands. 

We could consider a different command for local key or certificate later.

Reply:
We can fix the error and output messages first to make current usage clear enough to our users.

Reply:
Per the discuss in the community meeting on Apr 3, we agreed that change `notation key delete` to `notation key remove` and use `delete` as an alias of `remove`.